Formula/envoy.rb

```diff
@@ -60,6 +60,13 @@ def install
       --host_action_env=PATH=#{env_path}
     ]
 
+    if OS.linux? && OS.kernel_version < "4.6"
+      # Extension `tcp_stats` requires Linux headers >= 4.6
+      # (It's included by absolute path `#include </usr/include/linux/tcp.h>`)
+      # Disable it when the kernel version requirements not satisfied
+      args << "--//source/extensions/transport_sockets/tcp_stats:enabled=false"
+    end
+
```

Comment:
The CI container failed to detect the correct kernel version (Linux Kernel for Ubuntu 16.04 is 4.4.0).

Reply:
My guess is yes. But CC @Homebrew/linux

Reply:
Instead of looking at the kernel version, can we read the `tcp_stats.cc` file and make a decision based on that?

Reply:
https://github.com/envoyproxy/envoy/blob/release/v1.21/source/extensions/transport_sockets/tcp_stats/tcp_stats.cc#L12